moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -116,51 +115,36 @@ export default function TheMap({
   ]);
 
   const onMouseEnter = (e) => {
-    // hover states conflict! the first feature to reach hover state wins
-    // so set point radii and line widths accordingly
     setCursor("pointer");
-
-    e.features.forEach((feature) => {
-      if (feature.layer.id.includes("project")) {
-        // it's a project feature, so add to the hover tracking for sidebar interaction
-        setHoveredOnMapFeature(feature);
-      }
-      // mapRef.current?.setFeatureState(feature, { hover: true });
-    });
   };
 
   const onMouseLeave = (e) => {
     setCursor("grab");
-    setHoveredOnMapFeature(null);
-    // e.features.forEach((feature) => {
-    //   mapRef.current?.setFeatureState(feature, { hover: false });
-    //   // setHoveredOnMapFeatureId(null);
-    // });
   };
 
   const handleCreateOnClick = (e) => {
     const newDraftComponent = cloneDeep(draftComponent);
 
-    // Get the details we need to see if the feature is already in the draftComponent or not
+    /* Get the details we need to see if the feature is already in the draftComponent or not */
```

Comment:
is the switch to /* */ something prettier changed or is it personal preference? just curious